src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -129,20 +129,30 @@ public void removePerson(Person key) {
 
     /**
      * Returns true if tag already does exists.
-     * @param tag tag added
+     * @param tag The tag added.
      * @return True if tag exists. False if otherwise.
      */
     public boolean hasTag(Tag tag) {
         requireNonNull(tag);
         return tags.hasTag(tag);
     }
 
+    /**
+     * Adds the tag to the tagList.
+     * @param tag The tag to be added.
+     */
     public void addTag(Tag tag) {
         tags.addTagToList(tag);
     }
 
+    /**
+     * Changes the original tag to the new tag.
+     * @param oldTag The original tag
+     * @param newTag The new tag to replace the original tag.
```

Comment:
I believe that there should not be periods `.` at the end of `@param` javadocs.

Reply:
Ok. I made the changes.